python/sdk/merlin/model.py

```diff
@@ -794,6 +795,11 @@ def start(self):
 
         :return:
         """
+        # set the experiment in case there was other experiment set prior
+        # this assume the mlflow run and experiment were created during init
+        client = MlflowClient()
+        r = client.get_run(self._mlflow_run_id)
+        mlflow.set_experiment(experiment_id=r.info.experiment_id)
```

Comment:
@tiopramayudi if you can confirm that the run and experiment is always created prior to this during the model version init, that would be great 🙏

Reply:
yes usually it is creating during `new_model_version` function on model version